rules/aws/credential_access_aws_iam_assume_role_brute_force.toml

```diff
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2020/07/16"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/16"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a high number of failed attempts to assume an AWS IAM role. IAM roles are used to delegate access to users or
+services. An adversary may attempt to enumerate IAM roles in order to determine whether a role exists or not before
+attempting to assume or hijack the discovered role.
+"""
+index = ["filebeat-*"]
+language = "kuery"
+license = "Elastic License"
+name = "AWS IAM Assume Role Brute Force"
+references = [
+    "https://www.praetorian.com/blog/aws-iam-assume-role-vulnerabilities",
+    "https://rhinosecuritylabs.com/aws/assume-worst-aws-assume-role-enumeration/",
+]
+risk_score = 47
+rule_id = "ea248a02-bc47-4043-8e94-2885b19b2636"
+severity = "medium"
+tags = ["AWS", "Elastic"]
+type = "threshold"
+
+query = '''
+event.module:aws and event.dataset:aws.cloudtrail and event.provider:iam.amazonaws.com and event.action:UpdateAssumeRolePolicy and aws.cloudtrail.error_code:MalformedPolicyDocumentException and event.outcome:failure
```

Comment:
```suggestion
event.module:aws and event.dataset:aws.cloudtrail and 
  event.provider:iam.amazonaws.com and event.action:UpdateAssumeRolePolicy and 
  aws.cloudtrail.error_code:MalformedPolicyDocumentException and event.outcome:failure
```

I know we haven't formally standardized formatting yet, so feel free to ignore

Reply:
I dig it